src/Catty/Firebase/FirebaseCrashlyticsReporter.swift

```diff
@@ -22,27 +22,19 @@
 
 import Firebase
 
-extension AppDelegate {
+class FirebaseCrashlyticsReporter {
 
-    @objc open var analytics: Analytics.Type { Analytics.self }
-    @objc open var crashlytics: Crashlytics { Crashlytics.crashlytics() }
+    @objc open var crashlytics: Crashlytics
 
     var sendCrashReports: Bool { UserDefaults.standard.bool(forKey: kFirebaseSendCrashReports) }
     static let logNoValue = "No value"
 
-    @objc func setupFirebase() {
-        #if !DEBUG
-        FirebaseApp.configure()
-        setupAnalytics()
+    @objc init(crashlytics: Crashlytics) {
+        self.crashlytics = crashlytics
         setupCrashlytics()
-        #endif
     }
 
-    @objc public func setupAnalytics() {
-        analytics.setAnalyticsCollectionEnabled(true)
-    }
-
-    @objc public func setupCrashlytics() {
+    func setupCrashlytics() {
```

Comment:
Please do not make this function publicly available. Set access control to either `private` or `fileprivate`.